pkg/app/version.go

```diff
@@ -46,92 +40,18 @@ var CommandVersion = &cli.Command{
 			Value:   false,
 			Aliases: []string{"d"},
 		},
+		&formatter.FormatFlag,
 	},
+	Action: outputVersion,
 }
 
-func printVersion(ctx *cli.Context) error {
-	short := ctx.Bool("short")
-	detail := ctx.Bool("detail")
-	ver := version.GetVersion()
-	detailVer, err := getDetailedVersion(ctx)
-	fmt.Printf("envd: %s\n", ver)
-	if short {
-		return nil
-	}
-	fmt.Printf("  BuildDate: %s\n", ver.BuildDate)
-	fmt.Printf("  GitCommit: %s\n", ver.GitCommit)
-	fmt.Printf("  GitTreeState: %s\n", ver.GitTreeState)
-	if ver.GitTag != "" {
-		fmt.Printf("  GitTag: %s\n", ver.GitTag)
-	}
-	fmt.Printf("  GoVersion: %s\n", ver.GoVersion)
-	fmt.Printf("  Compiler: %s\n", ver.Compiler)
-	fmt.Printf("  Platform: %s\n", ver.Platform)
-	if detail {
-		if err != nil {
-			fmt.Printf("Error in getting details from Docker Server: %s\n", err)
-		} else {
-			fmt.Printf("  OSType: %s\n", detailVer.OSType)
-			if detailVer.OSVersion != "" {
-				fmt.Printf("  OSVersion: %s\n", detailVer.OSVersion)
-			}
-			fmt.Printf("  KernelVersion: %s\n", detailVer.KernelVersion)
-			fmt.Printf("  DockerHostVersion: %s\n", detailVer.DockerVersion)
-			fmt.Printf("  ContainerRuntimes: %s\n", detailVer.ContainerRuntimes)
-			fmt.Printf("  DefaultRuntime: %s\n", detailVer.DefaultRuntime)
-		}
+func outputVersion(clicontext *cli.Context) error {
+	format := clicontext.String("format")
+	switch format {
+	case "table":
+		return table.PrintVersion(clicontext)
```

Comment:
What's the current output of `envd version`?  Is it changed? I think our release pipeline depends on the output of `envd version --short`

Reply:
> What's the current output of `envd version`? Is it changed? I think our release pipeline depends on the output of `envd version --short`

It won't, `table` formatter is actually the original formatter. I could show an example output: 
```shell
xxx@amax:~/envd/bin$ ./envd version --short
envd: v0.2.5+e8126b1.dirty
```